Makefile

```diff
@@ -57,6 +57,7 @@ sparkly-clean: clean-apps clean-kazoo clean-release clean-deps
 clean: clean-kazoo
 	$(if $(wildcard *crash.dump), rm *crash.dump)
 	$(if $(wildcard scripts/log/*), rm -rf scripts/log/*)
+	$(if $(wildcard make/.deps.mk.*), rm make/.deps.mk.*)
```

Comment:
No, this will cause deps to be refreshed after every `clean` which is typically only done when cleaning kazoo-related code. `clean-deps` will cleanup the `.deps.mk.{HASH}` of the current `deps.mk`. Perhaps you could filter out the "valid" `.deps.mk.{HASH}` to clean up irrelevant versions?

Reply:
I agree I shouldn't remove .deps.mk. maybe we can add clean-all but shouldn't be delt with in this PR.